Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1

```diff
@@ -3361,38 +3361,50 @@ function Get-M365DSCExportContentForResource
     $Results = Format-M365DSCString -Properties $Results `
         -ResourceName $ResourceName
 
+    if ($Script:AllM365DscResources.Count -eq 0)
+    {
+        $Script:AllM365DscResources = Get-DscResource -Module 'Microsoft365Dsc'
+    }
+
     $primaryKey = ''
```

Comment:
This should be an array, or am I mistaken?

Reply:
$primaryKey is not an array right now and with my code I don't change that, it's still a regular string. The intention here is to select the best primary key since it may have more than one and $primaryKey should only include one of them, see next comment.